### ISSUE TITLE
rundeck_project_execution_duration_seconds return not the duration in secs

### ISSUE THREAD

**Issue by rdoering** [phsmith/rundeck_exporter#72]
|Key| Value|
|--|--|
|Deployment|Docker|
|version| 2.5.0|

With the PromQL `rundeck_project_execution_duration_seconds{execution_id="38774826"}` I get the result `100916` as show in detail here. 

```
rundeck_project_execution_duration_seconds{execution_id="38774826", execution_type="scheduled", instance="run-04", job="rundeck", job_group="...", job_id="70f1ef4a-a60b-4c2c-b1cb-daa827792a61", job_name="update_...", project_name="...", user="rundeck"}
100916
```

The API request `GET https://run-04/api/17/execution/38774826` returns

```
{
    "id": 38774826,
  ...
    "executionType": "scheduled",
    "user": "rundeck",
    "date-started": {
        "unixtime": 1686282983008,
        "date": "2023-06-09T03:56:23Z"
    },
    "date-ended": {
        "unixtime": 1686283095780,
        "date": "2023-06-09T03:58:15Z"
    },

I would expect a `rundeck_project_execution_duration_seconds` of `112` instead of `100916`.

The `/metrics/` says

```
# HELP rundeck_project_execution_duration_seconds Rundeck Project ProjectName Execution Duration
# TYPE rundeck_project_execution_duration_seconds gauge
```
Therefore I would expect a calculation like `(1686283095780 - 1686282983008) / 1000`.

**Comment by phsmith** [comment]
Hey @rdoering!
This issue has been fixed in version [v2.5.2](https://github.com/phsmith/rundeck_exporter/releases/tag/v2.5.2).
Can you update the exporter to this or the latest one and test again?

**Comment by rdoering** [comment]
Thanks for your fast reply. You are right, this issue is solved in 2.5.2.
Thank you very much.
